config.py

```diff
@@ -60,7 +60,7 @@ class ProductionConfig(Config):
     @classmethod
     def init_app(cls, app):
         Config.init_app(app)
-
+        assert os.environ.get('SECRET_KEY'), 'SECRET_KEY IS NOT SET!'
```

Comment:
So the way assert currently works is it checks if the SECRET_KEY exists. By the lines
```
 SECRET_KEY = os.environ.get('SECRET_KEY') or \
        'SjefBOa$1FgGco0SkfPO392qqH9%a492'
```
in config.py the secret_key will always "exist" within the Config class since it is either the environment variable or that random development key. The check currently in place for the `ProductionConfig` class firmly asserts that the SECRET_KEY has to be an environment variable...besides, the Config class pulls from the environment class to get the config variable anyway.